doc/uftrace-recv.md

```diff
@@ -27,6 +27,34 @@ Data will be sent using `uftrace-record` with -H/\--host option.
 :   Run given (shell) command as soon as receive data.  For example, one can run "uftrace replay" for received data.
 
 
+EXAMPLE
+=======
+This command can be used to receive data from remote.
+
+    # HOST 
+    $ uftrace recv -d recv_data --port 1234
```

Comment:
Please add the following setence:

"The uftrace recv should be run before record command sends data."